client/packages/system/src/Item/context/useUnitVariant.ts

```diff
@@ -132,6 +136,8 @@ export const useUnitVariant = (
     },
     numberOfPacksFromQuantity: totalQuantity =>
       NumUtils.round(totalQuantity / activeVariant.packSize, 2),
+    numberOfPacksToQuantity: numPacks =>
```

Comment:
I think we might need to rename these two to `TotalQuantity`, or does Quantity imply total ? What do you think ?

Reply:
[d3ae191](https://github.com/openmsupply/open-msupply/pull/2268/commits/d3ae1913a1b039e4e0c6d9410dc7ab5d3f8be3e4) (: made more explicit